crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -661,13 +658,14 @@ async fn store_record_zomes_to_invoke(
 
 async fn run_validation_callback(
     invocation: ValidateInvocation,
```

Comment:
I had moved the invocation creation from the parent method `validate_op` here, but for some reason the test became flaky based on app entries' zome indexes not being found, so I reverted the change. It was cleaner with the invocation being instantiated in here... ☹️